chromium_src/chrome/browser/DEPS

```diff
@@ -18,6 +18,7 @@ include_rules = [
   "+brave/components/brave_wallet",
   "+brave/components/brave_webtorrent/browser",
   "+brave/components/brave_webtorrent/grit",
+  "+brave/components/ai_chat_ui/common",
```

Comment:
please add this dep into `//brave/chromium_src/chrome/browser/ui` GN target.

Reply:
done